pyproject.toml

```diff
@@ -2,6 +2,69 @@
 requires = ["setuptools~=62.3", "wheel~=0.37.1"]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "arelle-release"
```

Comment:
Looks like version is required here:
```
DEFINITION:
    {
        "required": [
            "version"
        ]
    }

```

Reply:
Maybe we can consume `__version__` in `Version.py` by using the dynamic method?

Reply:
Damn, okay. `__version__` is the best option for now, go with that.